{
  "ID": "oval:org.debian:def:202131542",
  "Class": "vulnerability",
  "Metadata": {
    "Title": "CVE-2021-31542",
    "AffectedList": [
      {
        "Family": "unix",
        "Platform": "Debian GNU/Linux 9",
        "Product": "python-django"
      }
    ],
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2021-31542",
        "RefURL": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-31542"
      }
    ],
    "Description": "In Django 2.2 before 2.2.21, 3.1 before 3.1.9, and 3.2 before 3.2.1, MultiPartParser, UploadedFile, and FieldFile allowed directory traversal via uploaded files with suitably crafted file names.",
    "Debian": {
      "MoreInfo": ""
    }
  },
  "Criteria": {
    "Operator": "AND",
    "Criterias": [
      {
        "Operator": "OR",
        "Criterias": [
          {
            "Operator": "AND",
            "Criterias": null,
            "Criterions": [
              {
                "Negate": false,
                "TestRef": "oval:org.debian.oval:tst:2",
                "Comment": "all architecture"
              },
              {
                "Negate": false,
                "TestRef": "oval:org.debian.oval:tst:24080",
                "Comment": "python-django DPKG is earlier than 1:1.10.7-2+deb9u13"
              }
            ]
          }
        ],
        "Criterions": null
      }
    ],
    "Criterions": [
      {
        "Negate": false,
        "TestRef": "oval:org.debian.oval:tst:1",
        "Comment": "Debian 9 is installed"
      }
    ]
  }
}